download/zh-cn/download.md

```diff
@@ -7,12 +7,12 @@
 |:---:|:--:|:--:|:--:|
+| Sep. 30th, 2022 | 3.1.0 | Total Source Code | [[src]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.1.0/apache-dolphinscheduler-3.1.0-src.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/3.1.0/apache-dolphinscheduler-3.1.0-src.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/3.1.0/apache-dolphinscheduler-3.1.0-src.tar.gz.sha512) |
+|  | | Total Binary Distribution | [[bin]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.1.0/apache-dolphinscheduler-3.1.0-bin.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/3.1.0/apache-dolphinscheduler-3.1.0-bin.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/3.1.0/apache-dolphinscheduler-3.1.0-bin.tar.gz.sha512) |
 | Sep. 26th, 2022 | 3.0.1 | Total Source Code| [[src]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.0.1/apache-dolphinscheduler-3.0.1-src.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/3.0.1/apache-dolphinscheduler-3.0.1-src.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/3.0.1/apache-dolphinscheduler-3.0.1-src.tar.gz.sha512)|
 | | | Total Binary Distribution| [[bin]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.0.1/apache-dolphinscheduler-3.0.1-bin.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/3.0.1/apache-dolphinscheduler-3.0.1-bin.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/3.0.1/apache-dolphinscheduler-3.0.1-bin.tar.gz.sha512)|
 | Aug. 9th, 2022 | 3.0.0 | Total Source Code| [[src]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-src.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-src.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-src.tar.gz.sha512)|
 | | | Total Binary Distribution| [[bin]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz.sha512)|
-| Aug. 9th, 2022 | 3.0.0 | Total Source Code| [[src]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-src.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-src.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-src.tar.gz.sha512)|
-| | | Total Binary Distribution| [[bin]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/3.0.0/apache-dolphinscheduler-3.0.0-bin.tar.gz.sha512)|
```

Comment:
why do we remove the verison 3.0.0 download link?

Reply:
I see this part is repeated in the documentation.

Reply:
Yes, it's repeated.
![image](https://user-images.githubusercontent.com/11962619/193185677-a5915c1a-ef4b-48cf-99d0-f4694538e2f8.png)